cluster/helm/splice-global-domain/templates/mediator.yaml

```diff
@@ -125,6 +130,31 @@ spec:
       tolerations:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      volumes:
```

Comment:
I assume an empty volumes section is valid so wed on't need to move the Values.pvc above?

Reply:
if it works for the sequencer, I'm happy to keep it